recipes/dash-iconify/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "dash-iconify" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dash_iconify-{{ version }}.tar.gz
+  sha256: 564774be6b11b0ac3a8999b7137c3d17a1d351d69b673aa313c7228eacc9d143
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - dash
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
The pip checks for dash are broken, see [here](https://github.com/conda-forge/dash-mantine-components-feedstock/pull/9#discussion_r1466183308)